client/components/RelSongs.jsx

```diff
@@ -6,32 +6,32 @@ import styles from './RelSongsBox.css';
 function RelSongs({ song, key }) {
   return (
     <li key={key}>
-      <div className="gray">{song.artist}</div>
+      <div className="name">{song.artist}</div>
       <div className="songName">{song.songName}</div>
       <div><img alt={`${song.songName} albumPic`} src={`${song.albumPic}`} /></div>
       <div>
         <span className="gray">
           <FontAwesomeIcon icon="play" className="icon" />
           {' '}
```

Comment:
adding padding to the right and left of the span would be a more efficient way of handling this

Reply:
@sharmoo this would avoid filling up the file like this with empty quotes

Reply:
thanks!
I will do this!